src/services/farcasterPostReference.ts

```
@@ -0,0 +1,71 @@
+import { generateResponse } from "../openai/generateResponse";
+import { trackReplyPost } from "../stack/trackReplyPost";
+import { FELIZ_VIERNES_USERNAME } from "../consts";
+import { getLatestSleepEvent } from "../stack/getLatestSleepEvent";
+
+const SIGNER_PRIVATE_KEY = process.env.SIGNER_PRIVATE_KEY as Address;
+const APP_FID = Number(process.env.APP_FID);
+
+if (!SIGNER_PRIVATE_KEY) {
+  throw new Error("SIGNER_PRIVATE_KEY is not set");
+}
+if (!APP_FID || isNaN(APP_FID)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)